src/typeDefs/schema.graphql

```diff
@@ -46,22 +46,26 @@ type Facility {
   nameJa: String
   contact: Contact
   healthcareProfessionalIds: [String]
-  healthcareProfessionals: [HealthcareProfessional]
+  healthcareProfessionals: [HealthcareProfessional],
+  isDeleted: Boolean
```

Comment:
I think we decided against using the `isDeleted` flag 😅  we're going to use logging to track deleted items

Reply:
@ermish I don't think we clearly decided against the `isDeleted` flag? I think we just agreed to remove it from the Facility list. However, doing that alone wouldn't prevent the entry from appearing in the HealthcareProfessional query.

<img width="757" alt="Screenshot 2023-08-21 at 3 27 46 PM" src="https://github.com/ourjapanlife/findadoc-server/assets/31802656/19d486f6-a42b-4e40-b00d-6015e03c0945">

How will the logger know if a HealthcareProfessional is deleted if there isn't a property anywhere that says it is?

Reply:
You would have a log of the id being removed from a facility and then a log of the deleted healthcareprofessional entity

Reply:
@ermish Okay, I'll open a new issue that includes removing this field when the logger is added.